frontends/web/src/routes/new-settings/mobile-settings.tsx

```diff
@@ -32,19 +33,20 @@ import { TPagePropsWithSettingsTabs } from './type';
  **/
 export const MobileSettings = ({ deviceIDs, hasAccounts }: TPagePropsWithSettingsTabs) => {
   const isMobile = useMediaQuery('(max-width: 768px)');
+  const { t } = useTranslation();
   useEffect(() => {
     if (!isMobile) {
       route('/new-settings/appearance');
     }
   }, [isMobile]);
   return (
     <Main>
-      <Header title={<h2>Settings</h2>} />
+      <Header guideExists={false} title={<h2>{t('settings.title')}</h2>} />
```

Comment:
This PR also takes care some bits of code that apparently still not using locize for words, for example here ("Settings).